docs/src/main/asciidoc/chapters/administration.txt

```diff
@@ -1182,17 +1182,19 @@ conditions that vary from the environments individually tested in the various
 components. For example, Accumulo's use of HDFS includes many short block
 reads, which differs from the more common full file read used in most
 map/reduce applications. We have found that certain versions of Accumulo and
-Hadoop will include stability bugs that greatly affect overall stability. In
-our testing, Accumulo 1.6.2, Hadoop 2.6.0, and Zookeeper 3.4.6 resulted in a
-stable VM clusters that did not fail a month of testing, while Accumulo 1.6.1,
-Hadoop 2.5.1, and Zookeeper 3.4.5 had a mean time between failure of less than
-a week under heavy ingest and query load. We expect that results will vary with
-other configurations, and you should choose your software versions with that in
-mind.
-
-
-
-
-
-
-
+Hadoop will include stability bugs that greatly affect overall stability.
+Release notes typically contain information about versions used in
+release testing.
+
+The following table shows the Hadoop, Zookeeper and
+Thrift versions defined in the dependency section of the POM for building the
+artifacts.  
+
+|================================================
+|Accumulo |Hadoop |Zookeeper |Thrift
+|1.7            |2.2.0     |3.4.6          |0.9.1
+|1.8            |2.6.4     |3.4.6          |0.9.3
+|================================================
+
```

Comment:
> We could leave the table in the user manual and add the following sentences.

If we're adding clarifications, I would also suggest adding a note about what the "version specified in the pom" actually means. I'm positive that normal users don't understand all of the implications. What about a third point..

"The versions set in the pom for Hadoop and ZooKeeper are treated as stable releases which provide a large-breadth of compatibility across other versions of those components. They do not guarantee compatibility with any other versions because Hadoop and ZooKeeper do not provide any such guarantee."